osu.Game/Beatmaps/BeatmapDifficulty.cs

```diff
@@ -20,8 +20,11 @@ public class BeatmapDifficulty : EmbeddedObject, IBeatmapDifficultyInfo
         public float OverallDifficulty { get; set; } = IBeatmapDifficultyInfo.DEFAULT_DIFFICULTY;
         public float ApproachRate { get; set; } = IBeatmapDifficultyInfo.DEFAULT_DIFFICULTY;
 
-        public double SliderMultiplier { get; set; } = 1;
-        public double SliderTickRate { get; set; } = 1;
+        [Ignored]
+        public double BaseVelocity { get; set; } = 1;
+
+        [Ignored]
+        public double TickRate { get; set; } = 1;
```

Comment:
As the single test failure demonstrates, ignoring these is running head-first into #20883. These properties will get decoded and then promptly get discarded due to this logic:

https://github.com/ppy/osu/blob/88135a3be867a54ea5124718bd846f2a64fc7b78/osu.Game/Beatmaps/WorkingBeatmap.cs#L230-L231

Thus, making it effectively such that every single beatmap ever will have a base velocity and tick rate of 1.

Contrary to #23418, however, I'm not sure how to fix this, because I'm not sure where *else* these properties would live.

Reply:
I guess you could do whatever *this* hokey-pokey is but it just seems cursed to me...

```diff
diff --git a/osu.Game/Beatmaps/WorkingBeatmap.cs b/osu.Game/Beatmaps/WorkingBeatmap.cs
index ab790617bb..9f7bf60344 100644
--- a/osu.Game/Beatmaps/WorkingBeatmap.cs
+++ b/osu.Game/Beatmaps/WorkingBeatmap.cs
@@ -227,9 +227,14 @@ private Task<IBeatmap> loadBeatmapAsync()
                     // The original beatmap version needs to be preserved as the database doesn't contain it
                     BeatmapInfo.BeatmapVersion = b.BeatmapInfo.BeatmapVersion;
 
+                    var decodedBeatmapInfo = b.BeatmapInfo;
+
                     // Use the database-backed info for more up-to-date values (beatmap id, ranked status, etc)
                     b.BeatmapInfo = BeatmapInfo;
 
+                    // but keep the difficulty data I guess??
+                    b.BeatmapInfo.Difficulty = decodedBeatmapInfo.Difficulty;
+
                     return b;
                 }, loadCancellationSource.Token, TaskCreationOptions.LongRunning, TaskScheduler.Default);
             }
```